ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ReportJobStatus.tsx

```diff
@@ -104,10 +99,10 @@ function ReportJobStatus({
                 </FlexItem>
             </Flex>
         );
-    } else if (reportStatus.runState === 'SUCCESS') {
+    } else if (reportStatus.runState === 'DELIVERED') {
         statusColorClass = 'pf-u-success-color-100';
-        statusIcon = <CheckCircleIcon title="Report run was successful" />;
-        statusText = <p className="pf-u-success-color-100">Successful</p>;
+        statusIcon = <CheckCircleIcon title="Report was successfully delivered" />;
+        statusText = <p className="pf-u-success-color-100">Successfully delivered</p>;
```

Comment:
@zhenpesky I changed the text from `Successful` to `Successfully delivered` for when a report run successfully delivers a report to the user's email. Let me know if that's fine or if you still prefer what we had before. I think this is clearer

Reply:
sounds good